.test-infra/jenkins/dependency_check/dependency_check_report_generator.py

```diff
@@ -153,39 +154,6 @@ def prioritize_dependencies(deps, sdk_type):
   return high_priority_deps
 
 
-def compare_dependency_versions(curr_ver, latest_ver):
```

Comment:
remove this method from the report generator, put it into an independent file version_comparer.py that could be used by both report generator and jira manager.